source/user-manual/ruleset/mitre.rst

```diff
@@ -82,80 +80,97 @@ For this example, we require the following infrastructure:
++------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Endpoint         | Example description                                                                                                                                                                                                                  |
++==================+======================================================================================================================================================================================================================================+
+| **Wazuh server** | You can download the |WAZUH_OVA| or install it using the :doc:`installation guide </installation-guide/index>`.                                                                                                                      |
++------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Windows 11**   || We perform privilege escalation emulation attack on this endpoint.                                                                                                                                                                  |
+|                  || It is required to have a Wazuh agent installed and enrolled to the Wazuh server. To install the Wazuh agent, refer to the :doc:`Wazuh Windows installation guide </installation-guide/wazuh-agent/wazuh-agent-package-windows>`.    |
++------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Wazuh server
 ~~~~~~~~~~~~
+
 #. Append the following rules to the ``/var/ossec/etc/rules/local_rules.xml`` file:
 
    .. code-block:: xml
-      :emphasize-lines: 9,14
 
-      <group name="local,syslog,sshd,">
-
-        <rule id="100002" level="5">
-          <if_sid>5710</if_sid>
-          <description>sshd: authentication failed from $(srcip).</description>
-          <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
-        </rule>
+      <group name="windows,sysmon,privilege-escalation">
 
-        <rule id="100003" level="10" frequency="8" timeframe="120" ignore="60">
-          <if_matched_sid>100002</if_matched_sid>
-          <description>sshd: brute force trying to get access to the system.</description>
-          <same_srcip />
+        <rule id="160011" level="10">
```

Comment:
The custom rule ID must fall within the reserved range of 100000 to 120000: [https://documentation.wazuh.com/current/user-manual/ruleset/custom.html#adding-new-decoders-and-rules](https://www.google.com/url?q=https://documentation.wazuh.com/current/user-manual/ruleset/custom.html%23adding-new-decoders-and-rules&sa=D&source=docs&ust=1697443776324598&usg=AOvVaw0M-0o2u9amfW0DkUFPuxYw)